baremetal/src/include/ebbrt/GeneralPurposeAllocator.h

```diff
@@ -15,6 +15,40 @@
 #include <ebbrt/VMemAllocator.h>
 
 namespace ebbrt {
+
+// page fault handler for mapping in physical pages
+// to virtual pages on all cores
+class LargeRegionFaultHandler : public ebbrt::VMemAllocator::PageFaultHandler {
+  std::vector<ebbrt::Pfn> vecPfns;
+  uintptr_t sAddr;  // start addr of region
+  size_t ps = 2 * 1024 * 1024;  // large page size
```

Comment:
I'd rather not redefine this everywhere. Find one place to define large page size and use it here